### PR TITLE
[WebserverBundle] fixed the bug that caused that the webserver would …

### PR DESCRIPTION
**Pull request by SerkanYildiz**
Fixed the bug that caused that the webserver would start anyway even you said N.

| Q             | A
| ------------- | ---
| Branch?       | 3.3
| Bug fix?      | yes
| New feature?  | no
| BC breaks?    | no
| Deprecations? | no
| Tests pass?   | yes
| Fixed tickets | #24879 
| License       | MIT
| Doc PR        | /

### CODE REVIEW

File: src/Symfony/Bundle/WebServerBundle/Command/ServerStartCommand.php [hunk 1]
```diff
@@ -93,7 +93,7 @@ protected function execute(InputInterface $input, OutputInterface $output)
                 'You can either install it or use the "server:run" command instead.',
             ));
 
-            if ($io->ask('Do you want to execute <info>server:run</info> immediately? [yN] ', false)) {
+            if ($io->confirm('Do you want to execute <info>server:run</info> immediately? [yN] ', false)) {
```
Comment (chalasr) [inline src/Symfony/Bundle/WebServerBundle/Command/ServerStartCommand.php]: ` [yN]` should be removed

**Review by chalasr — CHANGES_REQUESTED** [review]

File: src/Symfony/Bundle/WebServerBundle/Command/ServerStartCommand.php (hunk 1 above)
Comment (SerkanYildiz) [reply src/Symfony/Bundle/WebServerBundle/Command/ServerStartCommand.php]: Removed the yN and committed. Thanks.

**Review by xabbuh — APPROVED** [review]

**Comment by chalasr** [comment]
FYI it seems like your git user is not recognized by github, you probably need to register a new email address on github from the output of `git config --global user.email`. See https://help.github.com/articles/why-are-my-commits-linked-to-the-wrong-user/#commits-are-not-linked-to-any-user

**Review by chalasr — APPROVED** [review]

**Review by ro0NL — APPROVED** [review]
Hm i recognize this change :) see https://github.com/symfony/symfony/pull/22317/files#diff-15a026db1c07d66e0572d8302ad7d717

to mergers: already in 3.4

**Comment by SerkanYildiz** [comment]
True. It is already fixed in 3.4 :)

**Review by fabpot — APPROVED** [review]

**Comment by fabpot** [comment]
Thank you @SerkanYildiz.

**Comment by chalasr** [comment]
Congrats for your first Symfony PR @SerkanYildiz!

**Comment by SerkanYildiz** [comment]
Thank you guys!

**Pull request closed without merging**